easybuild/easyconfigs/m/MATLAB/MATLAB-2025b.eb

```
@@ -0,0 +1,28 @@
+name = 'MATLAB'
+version = '2025b'
+
+homepage = 'https://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = SYSTEM
+
+sources = ['R%(version)s_Linux.iso']
+checksums = ['48af564ac03a01902ac6c692385bac017912d54bbb9b9e83505a8156c5ecd672']
+
+download_instructions = 'Download %s from mathworks.com' % sources[0]
```

```suggestion
download_instructions = f'Download {sources[0]} from mathworks.com'
```

lets keep the original fstring